generators/php/sdk/src/endpoint/request/WrappedEndpointRequestGenerator.ts

```diff
@@ -93,7 +96,49 @@ export class WrappedEndpointRequestGenerator extends FileGenerator<
                     clazz.addTrait(this.context.phpTypeMapper.convertToTraitClassReference(declaredTypeName));
                 }
             },
-            fileUpload: () => undefined,
+            fileUpload: (fileUpload) => {
+                for (const property of fileUpload.properties) {
+                    switch (property.type) {
+                        case "file": {
+                            // TODO: Clean this up.
```

Comment:
We can remove these if we don't plan to do it here - no problem to keep this as-is for now!